bin/bookkeeper

```diff
@@ -62,13 +58,25 @@ bookkeeper_help() {
     cat <<EOF
 Usage: bookkeeper <command>
 where command is one of:
+
+[service commands]
+
     bookie              Run a bookie server
     autorecovery        Run AutoRecovery service daemon
+    zookeeper           Run zookeeper server
```

Comment:
table service is run as part of bookie via configuration. the proxies part is not relocated to bookkeeper, we only moved the core dlog api (because that is more useful to most of bookkeeper users)

Reply:
sure